### PR TITLE
Annoyances Soical: pikicast.com

### PR DESCRIPTION
**Pull request by ghost**
### URLs
https://www.pikicast.com/
https://www.pikicast.com/#!/menu=landing&content_id=854912

### Screenshots
Screenshot-Copyright1
Screenshot-copyright
Screenshot

### CODE REVIEW

File: AnnoyancesFilter/sections/mobile-app.txt [hunk 1]
```diff
@@ -18,6 +18,7 @@ amp-reddit-com.cdn.ampproject.org##.AppSelectorModal__body
 reddit.com#$#.XPromoPopup { display: none!important; }
 reddit.com#$#body.scroll-disabled { overflow: visible!important; position: static!important; }
 !
+pikicast.com##.app-story-layer
```
Comment (Sergey-Lyapin) [inline AnnoyancesFilter/sections/mobile-app.txt:21]: we also need to remove the overlay and fix scrolling

**Review by Sergey-Lyapin — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/sections/mobile-app.txt (hunk 1 above)
Comment (piquark6046) [reply AnnoyancesFilter/sections/mobile-app.txt]: https://github.com/AdguardTeam/AdguardFilters/pull/82955/commits/fb8d6ca0c67396fea97cb70dec4e2266bd0bdca3

File: AnnoyancesFilter/sections/mobile-app.txt [hunk 2]
```diff
@@ -18,6 +18,7 @@ amp-reddit-com.cdn.ampproject.org##.AppSelectorModal__body
 reddit.com#$#.XPromoPopup { display: none!important; }
 reddit.com#$#body.scroll-disabled { overflow: visible!important; position: static!important; }
 !
+pikicast.com##.modal
```
Comment (zloyden) [inline AnnoyancesFilter/sections/mobile-app.txt:21]: Previously, this blocked the feed on the main page, now useful elements.
Try `||pikicast.com/js/view/layer/asset/layer.store.js`

**Review by zloyden — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/sections/mobile-app.txt (hunk 2 above)
Comment (piquark6046) [reply AnnoyancesFilter/sections/mobile-app.txt]: Okay. 
https://github.com/AdguardTeam/AdguardFilters/pull/82955/commits/c1c2fab53e9e58acd1f56c89fe0c602dcc4fc07a

**Review by zloyden — APPROVED** [review]

**Pull request merged**